plugins/net_plugin/net_plugin.cpp

```diff
@@ -920,7 +920,7 @@ namespace eosio {
       std::string                      remote_endpoint_ip     GUARDED_BY(conn_mtx);
       boost::asio::ip::address_v6::bytes_type remote_endpoint_ip_array GUARDED_BY(conn_mtx);
 
-      std::chrono::nanoseconds         connection_start_time{0};
+      std::atomic<std::chrono::nanoseconds>  connection_start_time;
```

Comment:
You can copy the one line PR description to here as comment too.

Reply:
The comment would quickly be out of date. I want to also access this in connection status which is http threads.